modules/EvseManager/EvseManager.cpp

```diff
@@ -1395,6 +1405,38 @@ static double get_cable_check_voltage(double ev_max_cpd, double evse_max_cpd) {
     return cable_check_voltage;
 }
 
+double EvseManager::get_over_voltage_threshold() {
+
+    float ev_max_voltage = 500.;
```

Comment:
Use double. It is assigned a double below (`ev_info.maximum_voltage_limit`), and assigned to a double afterwards (`double negotiated_max_voltage =`), no point in converting it around.

Reply:
ev_info.max_voltage_limit is float, and so is evse_caps.max_export_voltage_V. So std::min is with both float arguments and only one conversion to double takes place.

Reply:
Ah, I drew a wrong conclusion. I wasn't aware that `number` types in `manifest.yaml` are generated as config doubles, but that such types become floats in `types/*` (and presumably `interfaces/*`).